### ISSUE TITLE
SklearnTransformerWrapper error when not defining variables

### ISSUE THREAD

**Issue by adrianhartanto97** [solegalli/feature_engine#85]
**Describe the bug**
SklearnTransformerWrapper error when not defining variables (I'd like to transform all the features). 
But when defining any, then it will work.

**To Reproduce**
I demonstrate standardscaler like below image.

**Screenshots**
![image](https://user-images.githubusercontent.com/17286114/89046930-dec14b00-d377-11ea-81c8-c3ed9ece3579.png)

**Desktop (please complete the following information):**
 - OS: Windows 8.1
 - Browser: Chrome
 - Version: feature-engine 0.5.15, scikit-learn 0.23.1, pandas 1.0.5

Kindly help. Thanks.

**Comment by michalgromiec** [comment]
I can reproduce error. @solegalli - docs states that if variables parameter is not passed, numerical variables should be taken into account. Taht way we won't be able to run e.g. OneHotEncoder using this sklearn wrapper - is it intentional? If yes, I could push the fix.

**Comment by solegalli** [comment]
@michalgromiec thanks for pointing this out. I can see that I forgot to include the line: 
```
# find or check for numerical variables
self.variables = _find_numerical_variables(X, self.variables)
```

I probably didn't think this carefully. If we allow only numerical variables, then the SimpleImputer() with methods `frequent` or `arbitrary` and a string value can't be used. So probably limiting this to numerical variables is not a good idea.

Regarding the OneHotEncode, I anticipate a problem when returning the result, because the sklearn encoder returns a numpy array, so the SKlearnWrapper would have to have a workaround to add the feature names back to the dataframe.

So in short:
I think it would be best to allow all variables, not just numerical. 

And then, either add code to reconstitute the dataframe if the user passes the OneHotEncoder (probably the best option), or send a message that this transformer is not currently allowed (the fastest solution).

Thoughts?

**Comment by michalgromiec** [comment]
@solegalli seems logic. Just to be sure - transformer will transform selected features and will left others features untouched? 
I am in the middle of developing this change, including also OHE feature names.

**Comment by solegalli** [comment]
@michalgromiec yes, that is correct. TheOneHotEncoder from scikit-learn returns only the binary variables. I think we should concatenate these back to the original dataframe, which contains the remaining features untouched (that is, we would not drop the categorical variables that were encoded as the OneHotCategoricalEncoder from Feature-engine does.

**Comment by solegalli** [comment]
DONE!

**Comment by adrianhartanto97** [comment]
Thankyou :+1:
